### PR TITLE
[API Compatibility No.191] torch.nn.functional.poisson_nll_loss -> paddle.nn.functional.poisson_nll_loss -part

### PR DESCRIPTION
**Pull request by Aidenwu0209**
### PR Category
User Experience

### PR Types
New features

### Description
1. 为 `paddle.nn.functional.poisson_nll_loss` 添加 PyTorch 风格的参数别名支持
2. `target` 作为 `label` 的别名，`eps` 作为 `epsilon` 的别名
3. 新增参数别名单测（动态图、静态图、冲突错误测试）

### 是否引起精度变化
否

### CODE REVIEW

File: python/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -1626,6 +1612,8 @@ def poisson_nll_loss(
             Label tensor, random sampled from Poisson distribution :math:`label \sim \text{Poisson}(input)`.
             The shape of input tensor should be `(N, *)` or `(*)`, same shape as the input tensor.
             It's data type should be float16, bfloat16, float32, float64.
+            Alias Support: The parameter name ``target`` can be used as an alias for ``label``.
```
Comment (zhwesky2010) [inline python/paddle/nn/functional/loss.py]: 文档这个按标准的格式来写

File: python/paddle/nn/functional/loss.py [hunk 2]
```diff
@@ -1639,6 +1627,8 @@ def poisson_nll_loss(
          epsilon (float, optional):
             A small value to avoid evaluation of :math:`\log(0)` when `log_input`\ =\ ``False``. ``epsilon > 0``.
             Default: 1e-8.
+            Alias Support: The parameter name ``eps`` can be used as an alias for ``epsilon``.
```
Comment (zhwesky2010) [inline python/paddle/nn/functional/loss.py]: 文档这个按标准的格式来写

参考 https://github.com/PaddlePaddle/docs/blob/develop/docs/dev_guides/coding_agent_rules/api_compatibility/2-1-python-decorator.mdr

**Review by zhwesky2010 — DISMISSED** [review]
LGTM

**Comment by luotao1** [comment]
image
需要修复单侧失败

**Comment by zhwesky2010** [comment]
处理一下冲突，test_api_compatibility.py拆分了，添加到数字最大的test_api_compatibility_part3.py就可以

**Review by zhwesky2010 — APPROVED** [review]
